server/lobbyconnection.py

```diff
@@ -960,6 +961,17 @@ async def command_game_join(self, message):
         uuid = int(message["uid"])
         password = message.get("password")
 
+        async with self._db.acquire() as conn:
+            try:
+                await conn.execute(
+                    game_join_log.insert().values(
+                        player_id=player_id,
+                        game_id=uuid,
+                    )
+                )
+            except DBAPIError as e:
+                self._logger.error("writing to game join log failed")
```

Comment:
Using `.exception` from within an exception handler will automatically print the stack trace. No need to capture the exception with an `as e` syntax even.
```suggestion
                self._logger.exception("writing to game join log failed")
```